wlm/src/main/java/ai/djl/serving/wlm/Adapter.java

```diff
@@ -88,18 +87,18 @@ public static Adapter newInstance(
      * <p>This unregisters it in the wpc for new threads and all existing threads.
      *
      * @param wp the worker pool to remove the adapter from
-     * @param adapterName the adapter name
      * @param <I> the input type
      * @param <O> the output type
+     * @param adapterName the adapter name
      */
     public static <I, O> void unregister(WorkerPool<I, O> wp, String adapterName) {
         ModelInfo<I, O> wpc = (ModelInfo<I, O>) wp.getWpc();
         Adapter adapter = wpc.unregisterAdapter(adapterName);
-        // TODO Support worker adapter scheduling rather than register/unregister on all workers
-        for (WorkerGroup<I, O> wg : wp.getWorkerGroups().values()) {
-            for (WorkerThread<I, O> t : wg.getWorkers()) {
-                t.addConfigJob(adapter.unregisterJob(wpc, t.getThreadType()));
-            }
+        LinkedBlockingDeque<WorkerJob<I, O>> jobQueue = wp.getJobQueue();
+        WorkerJob<I, O> job = adapter.unregisterJob(wpc);
+        int numWorkers = wp.getWorkers().size();
+        for (int i = 0; i < numWorkers; i++) {
+            jobQueue.offer(job);
```

Comment:
This can not guarantee each worker pick up one job, one worker might pick up 2 jobs.

Reply:
Changed as discussed. Only support tp=max for lora case.